### PR TITLE
Backport Swift 2.0 changes to Swift 1.2

### PR DESCRIPTION
**Pull request by felixvisee**
This is a backport of the Swift 2.0 changes from #11 to Swift 1.2. The history looks a little weird..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,175 +1,69 @@
...
-```
-
-### Interaction
-
-An interaction holds up to five arguments. It's basically an equatable tuple of arguments:
-
-```swift
-let int: Interaction0 = interaction()
-```
+> to be written..
```
Comment (bckr) [inline README.md]: 😱

File: README.md [hunk 2]
```diff
@@ -1,175 +1,69 @@
...
-    .Uppercase(value("input")),
-    .Concat(value("first"), value("second"))
-]))
-```
+Please check out the [source](https://github.com/rheinfabrik/Dobby/tree/swift-1.2/Dobby) and [tests](https://github.com/rheinfabrik/Dobby/tree/swift-1.2/DobbyTests) for further documentation.
 
 ## About
 
-Dobby was built by [Rheinfabrik](http://www.rheinfabrik.de) 🏭
+Dobby was built at [Rheinfabrik](http://www.rheinfabrik.de) 🏭
```
Comment (bckr) [inline README.md]: dobby

Comment (felixvisee) [inline README.md]: > > Dobby has no master.
> 
> Dobby was born at [Rheinfabrik](http://www.rheinfabrik.de) 🏭

Comment (bckr) [inline README.md]: n5msqe7

**Comment by bckr** [comment]
Nice! Thanks for the backporting work! :+1:

**Comment by DerLobi** [comment]
nice indeed!

**Pull request merged**